lib/fetchData.js

```diff
@@ -9,34 +9,46 @@ let data;
 
 const fetchData = () => {
   return Promise.all([
-    fetch('https://raw.githubusercontent.com/resin-io/etcher/master/docs/CLI-INSTALLATION.md')
-    .then(res => ( res.text() )),
-    fetch('https://raw.githubusercontent.com/resin-io/etcher/master/CHANGELOG.md')
-    .then(res => ( res.text() )),
-    S3(locals.s3Bucket, locals.title.toLowerCase())
+    fetch(
+      'https://raw.githubusercontent.com/resin-io/etcher/master/docs/CLI-INSTALLATION.md'
+    ).then(res => res.text()),
+    fetch(
+      'https://raw.githubusercontent.com/resin-io/etcher/master/CHANGELOG.md'
```

Comment:
Maybe we can store the URLs as constants? I imagine this should make these lines easier to read, e.g. `fetch(ETCHER_RAW_CHANGELOG_URL)` or something like that.